tests/e2e/specs/ccpa-revisit-popup-direct.spec.ts

```
@@ -0,0 +1,197 @@
+                array( '%d' )
+              );
+              delete_option( 'faz_banner_template' );
+            }
+          `);
+        }
+      } catch (e) {
+        // CodeRabbit#1: do NOT silence cleanup failures. A failed
+        // rollback leaves the banner law on 'ccpa' and the NEXT test
+        // (the GDPR regression below) inherits a polluted DB, flaking
+        // intermittently. Surface the failure so the test reports it.
+        throw new Error(`CCPA cleanup rollback failed: ${String(e)}`);
```

Biome (2.4.15)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)